examples/model_selection/grid_search_digits.py

```diff
@@ -60,9 +60,11 @@
     print()
     print("Grid scores on development set:")
     print()
-    for params, mean_score, scores in clf.grid_scores_:
+    means = clf.results_['test_%s_weighted_mean' % score]
```

Comment:
Without looking at the script, I was reading this as `test recall weighted_mean`, which again means that it might be better to push the mean before the scorer name